### DISCUSSION TITLE
Updating Icon of TrackNode...

### DISCUSSION THREAD

**Discussion by keince** [pelicanmapping/osgearth#1933] [Support Q&A]
I know this question has been asked before (back in 2015) but I will ask again...

Does the current osgEarth allow for updating the Icon of a TrackNode during runtime?

Thx

**Comment by gwaldron** [comment]
I have not looked at TrackNode in ages, but I suspect the answer is no. To achieve this you would (at least) need to add to the API and allow the node to re-compile its geometry.
